classes/class-wc-connect-settings-view.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Settings_View' ) ) {
+
+	class WC_Connect_Settings_View {
```

Comment:
The `Packages Manager` is going to be included in this page? If this is only for payment method selection, then the `WC_Connect_Settings_View` name is too generic.